### ISSUE TITLE
MAX_POPULATION prevents spawning of civs/ empty villages

### ISSUE THREAD

**Issue by CurePindal** [antoinebidault/DynamicCivilWar#65]
In a lot of cases the MAX_POPULATION value prevents spawning of new civilians when you travel through a few villages.
You reach new villages and they are empty. And nobody is spawning. So often theres no civs for intel and no village chief to expand your territory.

For this case there should be a kind of "fallback code" that removes other civilians (maybe those that are the furthest away?) to spawn new ones around you.

**Comment by antoinebidault** [comment]
I am aware of this issue. There is some kind of spawning system inertia... When you are in a sector with many buildings, the spawning system is overloaded very quickly. I have no idea on how to handle properly this fallback without a performance breaking solution. Not easy, but maybe I could do some garbage collection of items spawned in the most distant villages already populated... I need an additional param : the distance of "forced spawning".

**Comment by CurePindal** [comment]
Yes, I think an expanded gargbage collection/ AI cacheing system could easily solve it.
Btw, on my last solo playthrough on a listenserver I just raised the civ limits and had no more "empty village" issues. Maybe it was the changed values or I was just moving very slowly, not sure yet.
